src/ErrorBoundary/ErrorBoundary.tsx

```diff
@@ -97,7 +91,10 @@ class ErrorBoundary extends React.Component<
             return children;
         }
 
-        const appDisplayName = appName || packageJson().displayName;
+        const appDisplayName =
+            appName ||
+            ((packageJson() as AppPackageJson).displayName ??
+                packageJson()?.name);
```

Comment:
Isn't this just an x OR y OR z? Why are the last 2 options treated differently.

```suggestion
            appName ||
            (packageJson() as AppPackageJson).displayName ||
            packageJson()?.name;
```